ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -85,16 +85,20 @@ function pr-nnstreamer-ubuntu-apptest-run-queue() {
 
     # Download tensorflow-lite model file and labels.
     mkdir tflite_model
-    cd tflite_model    
-    wget https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_1.0_224_quant/mobilenet_v1_1.0_224_quant.tflite
+    cd tflite_model
+    echo -e "" >> ../../../report/pr-nnstreamer-apptest.log
+    echo -e "[DEBUG] Starting wget tflite model..." >> ../../../report/pr-nnstreamer-apptest.log
+    wget https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/mobilenet_v1_1.0_224_quant.tflite &>> ../../../report/pr-nnstreamer-apptest.log
     result+=$?
-    wget https://raw.githubusercontent.com/nnsuite/testcases/master/DeepLearningModels/tensorflow-lite/Mobilenet_1.0_224_quant/labels.txt
+    echo -e "" >> ../../../report/pr-nnstreamer-apptest.log
+    echo -e "[DEBUG] Starting wget tflite label..." >> ../../../report/pr-nnstreamer-apptest.log
+    wget https://raw.githubusercontent.com/nnsuite/testcases/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt &>> ../../../report/pr-nnstreamer-apptest.log
```

Comment:
Why do we have to use '&" instead of 0 or 1?

Reply:
Right, But, please improve a output style as follows. 

`...... 2>pr-nnstreamer-apptest-error.log 1>pr-nnstreamer-apptest-output.log`